packages/Molecules/Card/CardCarousel.vue

```diff
@@ -0,0 +1,210 @@
+<template>
+  <div class="rpl-carousel">
+    <h2 v-if="title" class="rpl-carousel__title">{{ title }}</h2>
+    <div class="rpl-carousel__slider">
+      <carousel
+        :perPage="slidesPerPage"
+        :navigateTo="navTo"
+        :mouseDrag="false"
+        :paginationEnabled="false"
+      >
+        <slide v-for="(item, index) in cards" :key="index" class="rpl-carousel__slide">
+          <component :is="item.name" v-bind="item.data"></component>
+        </slide>
+      </carousel>
+      <div class="rpl-carousel__navigation">
+        <button class="rpl-carousel__navigation-button rpl-carousel__navigation-button--prev" role="button" @click="prevSlide" :disabled="prevDisabled" :aria-label="previousLabel">
+          <rpl-icon :symbol="prevIcon" :color="iconColor(prevDisabled)" size="1.6" />
+        </button>
+        <button class="rpl-carousel__navigation-button rpl-carousel__navigation-button--next" role="button" @click="nextSlide" :disabled="nextDisabled" :aria-label="nextLabel">
+          <rpl-icon :symbol="nextIcon" :color="iconColor(nextDisabled)" size="1.6" />
+        </button>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import breakpoint from '@dpc-sdp/ripple-global/mixins/breakpoint'
+import RplIcon from '@dpc-sdp/ripple-icon'
+import { Carousel, Slide } from 'vue-carousel'
+
+import RplCardPromotion from './CardPromotion.vue'
+import RplCardEvent from './CardEvent.vue'
+import RplCardKeydates from './CardKeydates.vue'
+// TODO: Add future card components
+
+export default {
+  name: 'RplCarousel',
```

Comment:
Same, the name need to be `RplCardCarousel`.

Reply:
Fixed.